services/friendbot/internal/friendbot_test.go

```diff
@@ -17,7 +17,7 @@ func TestFriendbot_makeTx(t *testing.T) {
 		sequence:        2,
 	}
 
-	txn, err := fb.makeTx("GDJIN6W6PLTPKLLM57UW65ZH4BITUXUMYQHIMAZFYXF45PZVAWDBI77Z", fb.StartingBalance)
+	txn, err := fb.makeTx("GDJIN6W6PLTPKLLM57UW65ZH4BITUXUMYQHIMAZFYXF45PZVAWDBI77Z", fb.StartingBalance, false)
```

Comment:
can you add a test where the argument is `true` instead of false?

Reply:
As I don't know what bug the regression test is testing for, I don't see a need to verify that a payment tx can be properly constructed.